src/account/operations/participation/event.rs

```diff
@@ -65,16 +67,14 @@ impl AccountHandle {
     }
 
     /// Retrieves information for all registered participation events.
-    pub async fn get_participation_events(&self) -> crate::Result<Vec<ParticipationEvent>> {
-        Ok(self
-            .storage_manager
+    pub async fn get_participation_events(
```

Comment:
Also in the main changelog pls

Reply:
But we didn't do a new release since it was added?

Reply:
Ah yeah right, forget about me, I'm too used to ask for a changelog now :trollface: